extensions/amp-analytics/analytics-vars.md

```diff
@@ -473,4 +473,11 @@ Provides the total time from the time page was loaded to the time a ping was sen
 
 Provides the total time for which the element has met the visiblitySpec conditions at time this ping is sent.
 
+### fromSlide
+
+Provides the slide from which the traversal happens. The value is either taken from `data-slide-id` attribute of the slide when present , else it represents the index of the slide (starting from 0).
```

Comment:
Add "the" before "from". Also, remove extra space between "present" and the comma.
(do same below)